src/main/resources/templates/chart_report.vm

```diff
@@ -251,8 +251,20 @@ $webResourceManager.requireResource("org.everit.jira.timetracker.plugin:jttp-res
 						</tr>
 						<tr>
 							<td>
-								<input id="search"  tabindex="3" type="submit" name="search" value="$i18n.getText('plugin.search')" class="button" />
+								<input id="search"  tabindex="3" type="submit" name="search" value="$i18n.getText('plugin.search')" class="button" 
+								  onclick="searchOnClick();"/>
 							</td>
+							<script>
```

Comment:
cant you take this snippet into a standalone js file? The function would take parameters like 'MyChart'.

Having inline scripts can cause real issues and should be avoided. E.g.: If you refactor later and ajaxify the screens, you can have issues with the inline scripts.